src/Avalonia.Controls/Platform/IScreenImpl.cs

```diff
@@ -1,11 +1,19 @@
 ﻿using System.Collections.Generic;
 
+#nullable enable
+
 namespace Avalonia.Platform
 {
     public interface IScreenImpl
     {
         int ScreenCount { get; }
 
         IReadOnlyList<Screen> AllScreens { get; }
+
+        Screen? ScreenFromWindow(IWindowBaseImpl window);
+
+        Screen? ScreenFromPoint(PixelPoint point);
```

Comment:
Can't be implemented with Wayland

Reply:
Then it should return `null` i guess?

Reply:
public Screen ScreenFromPoint(PixelPoint point)
        {
            return ScreenHelper.ScreenFromPoint(point, AllScreens);
        }

Reply:
even with this?